File: test/fixtures/python/get_user_agent_alias.py
import requests

headers = {
    'x-msisdn': 'XXXXXXXXXXXXX',
    'user-agent': 'Mozilla Android6.1',
}

params = {
    'p': '5',
    'pub': 'testmovie',
    'tkn': '817263812',
}

response = requests.get('http://localhost:28139/vc/moviesmagic', headers=headers, params=params)

File: test/fixtures/r/get_user_agent_alias.r
library(httr)

headers = c(
  `x-msisdn` = "XXXXXXXXXXXXX",
  `user-agent` = "Mozilla Android6.1"
)

params = list(
  p = "5",
  pub = "testmovie",
  tkn = "817263812"
)

res <- httr::GET(url = "http://localhost:28139/vc/moviesmagic", httr::add_headers(.headers=headers), query = params)
